scripts/clean_yarn_cache_selectively.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
This script is not used by any other scripts or by CI. Just here for running manually if you'd like to clear all packages from the yarn cache.